api.planx.uk/saveAndReturn/utils.ts

```diff
@@ -120,7 +110,7 @@ const sendSingleApplicationEmail = async (
       reference: null,
       emailReplyToId: team.notifyPersonalisation.emailReplyToId,
     };
-    const firstSave = !session.has_user_saved;
+    const firstSave = !session.hasUserSaved;
```

Comment:
Previously, this would always evaluate to `true` as `session.has_user_saved` was `undefined`.

However, it did not set up duplicate events in Hasura on each call. This is because the event trigger watching the `lowcal_sessions.has_user_saved` column wasn't being fired as the value never changed - and overwrite from `true` to `true` does not count as a change which is good to be aware of. 

I'm not exactly clear on where the responsibility for this is - potentially the GraphQL client wasn't performing a redundant mutation, or Hasura was ignoring one.

Reply:
So it would still send an email on the first save, the code being accidentally correct?

Reply:
Exactly 😬